src/integration/transaction/transaction.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from '@nestjs/common';
+import { AinModule } from 'src/blockchain/ain/ain.module';
+import { BlockchainModule } from 'src/blockchain/blockchain.module';
+import { SharedModule } from 'src/shared/shared.module';
+import { TransactionController } from './api/controllers/transaction.controller';
+import { TransactionExecutionService } from './application/services/transaction-execution.service';
+import { TransactionService } from './application/services/transaction.service';
+
+@Module({
+  imports: [SharedModule, AinModule, BlockchainModule],
```

Comment:
BlockchainModule should be sufficient (no AinModule)

Reply:
fixed in #42